circle.yml

```diff
@@ -5,38 +5,21 @@ deployment:
   staging:
     branch: master
     commands:
-      - |
-        cat >~/.netrc <<EOF
-        machine api.heroku.com
-          login $HEROKU_EMAIL
-          password $HEROKU_TOKEN
-        machine git.heroku.com
-          login $HEROKU_EMAIL
-          password $HEROKU_TOKEN
-        EOF
-      - chmod 600 ~/.netrc
-      - "[[ ! -s \"$(git rev-parse --git-dir)/shallow\" ]] || git fetch --unshallow"
+      - sh scripts/heroku_login.sh
       - DEPLOY_ENV=staging yarn deploy
   production:
     branch: release
     commands:
-      - |
-        cat >~/.netrc <<EOF
-        machine api.heroku.com
-          login $HEROKU_EMAIL
-          password $HEROKU_TOKEN
-        machine git.heroku.com
-          login $HEROKU_EMAIL
-          password $HEROKU_TOKEN
-        EOF
-      - chmod 600 ~/.netrc
-      - "[[ ! -s \"$(git rev-parse --git-dir)/shallow\" ]] || git fetch --unshallow"
+      - sh scripts/heroku_login.sh
       - DEPLOY_ENV=production yarn deploy
 dependencies:
   override:
+    - rm -rf node_modules/electron
+    - rm -rf node_modules/electron-download
+    - npm install electron
     - yarn install
 test:
   override:
     - "gem install danger --version '~> 4.0' && danger"
-    - case $CIRCLE_NODE_INDEX in 0) yarn mocha test/lib/* && yarn mocha $(find desktop/test -name '*.coffee') && yarn mocha $(find mobile/test -name '*.coffee') ;; 1) yarn mocha $(find desktop/components/*/test -name '*.coffee') && yarn mocha $(find desktop/components/**/*/test -name '*.coffee') ;; 2) yarn mocha $(find desktop/apps/*/test -name '*.coffee') && yarn mocha $(find desktop/apps/*/test -name '*.js') && yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee') ;; 3) yarn mocha $(find mobile/components/*/test -name '*.coffee') && yarn mocha $(find mobile/components/**/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/**/*/test -name '*.coffee'); esac:
+    - yarn test:
```

Comment:
I PR creeped this by cleaning up our circle.yml to always get a fresh install of Electron (hopefully circumventing the cache issue we've run into all the time). It also expands on scripts/test.sh so we can use that same script for Circle and local `yarn test`.